vnext/src/RNTester/js/examples-win/Theming/ThemingExample.windows.tsx

```diff
@@ -6,23 +6,23 @@
 
 import React = require('react');
 import {Text, View, Button} from 'react-native';
-import {AppTheme} from '../../../../Libraries/AppTheme/AppTheme';
-import {IAppThemeChangedEvent} from '../../../../Libraries/AppTheme/AppThemeTypes';
+import {Appearance} from 'react-native';
```

Comment:
Appearance already has its own test page. Seems like we should just remove this one altogether.